provider/k8s/k8s.go

```diff
@@ -151,6 +153,8 @@ func FromEnv() (*Provider, error) {
 		ConvoxDomainTLSCertDisable: os.Getenv("CONVOX_DOMAIN_TLS_CERT_DISABLE") == "true",
 		CertManagerClient:          cm,
 		DiscoveryClient:            kc.Discovery(),
+		DockerUsername: 			common.CoalesceString(os.Getenv("DOCKER_HUB_USERNAME"),""),
+		DockerPassword: 			common.CoalesceString(os.Getenv("DOCKER_HUB_PASSWORD"),""),
```

Comment:
common.CoalesceString is not needed here
currently it is like if the value is empty then set empty.
fix this please

Reply:
Added the changes...